Sources/Sentry/SentryBinaryImageCache.m

```diff
@@ -99,6 +99,19 @@ - (NSInteger)indexOfImage:(uint64_t)address
     return -1; // Address not found
 }
 
+- (nullable NSString *)pathForImage:(NSString *)binaryName
+{
+    @synchronized(self) {
+        for (SentryBinaryImageInfo *info in _cache) {
```

Comment:
`l`: This is `n*m` where `n` is the number of `inAppIncludes` and `m` is the number of binary images. Having a large number of inAppIncludes in a large app could take some time. But I guess it should be fine for now, as we compare by prefix, so building a dictionary for faster lookup to get this to m is a bit tricky and not worth the effort right now, IMO. WDTY?

Reply:
I don't think we need the refactoring right now, this is called once when the app starts, even with 500 libs, and 10 items in the `inAppInclude` properties, this only takes microseconds.

If we ever use this function regularly during the session we may consider improving it.